client/lib/signup/step-actions.js

```diff
@@ -205,6 +205,11 @@ export function createSiteWithCart( callback, dependencies, stepData, reduxStore
 		newSiteParams.options.site_segment = 1;
 	}
 
+	// Provide siteTitle from Gutenboarding
+	if ( 'frankenflow' === lastKnownFlow ) {
```

Comment:
Is `frankenflow` going to be ever visible in the route fragment, or it is just a placeholder for now?

Only asking as it might cause some confusion if users see `frankenflow` in the URL :)

Reply:
Since this whole concept is not made for public, I assume the users would be `frankenflow`-aware  🙈But I'm happy to change it to `pre-launch` or other name if needed.

Reply:
This will all be cleaned up, fortunately `frankenflow` is easy to find and replace 🙂